src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1607,11 +1607,10 @@ describe('ReactDOMComponent', () => {
       ReactDOMServer.renderToString(<input type="text" onclick="1" />);
       expectDev(console.error.calls.count()).toBe(2);
       expect(normalizeCodeLocInfo(console.error.calls.argsFor(0)[0])).toBe(
-        'Warning: Unknown DOM property class. Did you mean className?\n    in div (at **)',
+        'Warning: Unknown DOM property class. Did you mean className?',
       );
       expect(normalizeCodeLocInfo(console.error.calls.argsFor(1)[0])).toBe(
-        'Warning: Unknown event handler property onclick. Did you mean ' +
-          '`onClick`?\n    in input (at **)',
+        'Warning: Unknown event handler property onclick. Did you mean `onClick`?',
```

Comment:
Ah okay. I don't really mind since it's easier to debug things on the client anyway.